tools/cli/go-whisk-cli/commands/commands.go

```diff
@@ -60,6 +60,7 @@ func setupClientConfig(cmd *cobra.Command, args []string) (error){
         Version:    Properties.APIVersion,
         Insecure:   flags.global.insecure,
         Host:       Properties.APIHost,
+        UserAgent:  "OpenWhisk-CLI/" + Properties.CLIVersion,
```

Comment:
@drcariel - define the value used by the cli in an exported string variable, then reference it here